README.rst

```diff
@@ -109,7 +109,9 @@ Make sure you have installed the following dependencies:
 
 On debian based systems this can generally be done by running::
 
-    sudo apt-get install -y build-essential libdumbnet-dev libpcap-dev libgeoip-dev libffi-dev python-dev python-pip
+    sudo apt-get install -y build-essential libdumbnet-dev libpcap-dev \
+        libgeoip-dev libffi-dev python-dev python-pip libssl-dev libyaml-dev \
+        libpcap0.8-dev libgmp-dev tor tor-geoipdb
```

Comment:
The follows deps are actually not needed: 
* libyaml-dev
* libpcap0.8-dev
* libgmp-dev
* tor-geoipdb

I am also not 100% sure about libssl-dev.

It would also be good to keep the command on one line to facilitate copy paste.